JS/test/airport.spec.js

```diff
@@ -20,38 +20,38 @@ describe('My Test', () => {
         new PassengerPlane('Embraer 190', 870, 8100, 30800, 64),
         new PassengerPlane('Sukhoi Superjet 100', 870, 11500, 50500, 140),
         new PassengerPlane('Bombardier CS300', 920, 11000, 60700, 196),
-        new MilitaryPlane('B-1B Lancer', 1050, 21000, 80000, MilitaryType.BOMBER),
-        new MilitaryPlane('B-2 Spirit', 1030, 22000, 70000, MilitaryType.BOMBER),
-        new MilitaryPlane('B-52 Stratofortress', 1000, 20000, 80000, MilitaryType.BOMBER),
-        new MilitaryPlane('F-15', 1500, 12000, 10000, MilitaryType.FIGHTER),
-        new MilitaryPlane('F-22', 1550, 13000, 11000, MilitaryType.FIGHTER),
-        new MilitaryPlane('C-130 Hercules', 650, 5000, 110000, MilitaryType.TRANSPORT),
-        new experimentalPlane("Bell X-14", 277, 482, 500, ExperimentalTypes.HIGH_ALTITUDE, ClassificationLevel.SECRET),
-        new experimentalPlane("Ryan X-13 Vertijet", 560, 307, 500, ExperimentalTypes.VTOL, ClassificationLevel.TOP_SECRET)
+        new MilitaryPlane('B-1B Lancer', 1050, 21000, 80000, MilitaryType.Bomber),
+        new MilitaryPlane('B-2 Spirit', 1030, 22000, 70000, MilitaryType.Bomber),
+        new MilitaryPlane('B-52 Stratofortress', 1000, 20000, 80000, MilitaryType.Bomber),
+        new MilitaryPlane('F-15', 1500, 12000, 10000, MilitaryType.Fighter),
+        new MilitaryPlane('F-22', 1550, 13000, 11000, MilitaryType.Fighter),
+        new MilitaryPlane('C-130 Hercules', 650, 5000, 110000, MilitaryType.Transport),
+        new experimentalPlane("Bell X-14", 277, 482, 500, ExperimentalTypes.highAltitude, ClassificationLevel.Secret),
+        new experimentalPlane("Ryan X-13 Vertijet", 560, 307, 500, ExperimentalTypes.VTOL, ClassificationLevel.topSecret)
     ];
     let planeWithMaxPassengerCapacity = new PassengerPlane('Boeing-747', 980, 16100, 70500, 242);
 
-    it('should have military Planes with transport type', () => {
+    it('list of military planes with transport type', () => {
         let airport = new Airport(planes);
         let transportMilitaryPlanes = airport.getTransportMilitaryPlanes();
         let flag = false;
         for (let militaryPlane of transportMilitaryPlanes) {
```

Comment:
Please correct this code. There is an additional logic implemented. It can be written shorter (almost in one line).

Reply:
Done